README.md

```diff
@@ -1,20 +1,30 @@
-BanyanDB
-============
+# BanyanDB
 
 ![](https://github.com/apache/skywalking-banyandb/workflows/Build/badge.svg?branch=main)
 
 ![](./assets/banyandb_small.jpg)
 
-BanyanDB, as an observability database, aims to ingest, analyze and store Metrics, Tracing and Logging data. 
-It's designed to handle observability data generated by Observability platform and APM system, like [Apache SkyWalking](https://github.com/apache/skywalking) etc. 
+BanyanDB, as an observability database, aims to ingest, analyze and store Metrics, Tracing and Logging data.
+It's designed to handle observability data generated by Observability platform and APM system, like [Apache SkyWalking](https://github.com/apache/skywalking) etc.
 
-# Resource
+## Introduction
 
- - [Data model design](https://docs.google.com/document/d/1XFzHWXSWyIC7AVNIx0ifQ0eJAJYfoPO-2j_lgbl9zu8/edit?usp=sharing)
+BanyanDB, as an observability database, aims to ingest, analyze and store Metrics, Tracing, and Logging data. It's designed to handle observability data generated by Apache SkyWalking. Before BanyanDB emerges, the Databases that SkyWalking adopted are not ideal for the APM data model, especially for saving tracing and logging data. Consequently, There’s room to improve the performance and resource usage based on the nature of SkyWalking data patterns.
 
+The database research community usually uses [RUM conjecture](http://daslab.seas.harvard.edu/rum-conjecture/) to describe how a database access data. In this talk, I will share the details about BanyanDB based on this conjecture, explaining why BanyanDB is more effective and reliable than other storage layers.
+
+## Documents
+
+[Documents](./docs/)
+
+## Resource
+
+- [Data model design](https://docs.google.com/document/d/1XFzHWXSWyIC7AVNIx0ifQ0eJAJYfoPO-2j_lgbl9zu8/edit?usp=sharing)
```

Comment:
Let's keep the valuable content of this google doc in the markdown as official, and remove this link permanently. I don't want to misguide new contributors/users, and read a version has been changed.

Reply:
https://github.com/apache/skywalking/issues/9016 will create update-date documents